test/UnitTest.cs

```diff
@@ -140,6 +140,23 @@ bool FindInLevel(List<PhysicNode> curlevel, string pattern, out List<PhysicNode>
             }
             return true;
         }
+
+        public static void ClearTableStatsInCatalog(List<String> tabNameList)
+        {
+            foreach (String tabName in tabNameList)
+            {
+                try
```

Comment:
please remove try/catch here - I'd rather to see a more obvious panic error message instead of hide the errors.

Reply:
got it.